config/application.rb

```diff
@@ -134,5 +134,18 @@ class Application < Rails::Application
     # These files do not contain auto-loaded ruby classes,
     #   they are loaded through app/sidekiq/education_form/forms/base.rb
     Rails.autoloaders.main.ignore(Rails.root.join('app', 'sidekiq', 'education_form', 'templates', '1990-disclosure'))
+
+    # ENVs from Parent Helm Chart are all lowercase
+    # to keep with the convention they should all
+    # be upcased ENV['example'] => ENV['EXAMPLE]
+    # In order to not disrupt other ENVs. This only
+    # adds an upcased version.
```

Comment:
Suggestion to make it easier to understand on the first read. 
```suggestion
    # ENVs from Parent Helm Chart are all lowercase.
    # To keep with the convention, they should all
    # be upcased in order to not disrupt other ENVs.
    # This code adds an upcased version
    # ENV['example'] => ENV['EXAMPLE]
```